docs/release-notes.md

```diff
@@ -9,6 +9,26 @@ meta:
 ---
 # Release Notes
 
+## 5.0.0
+### SAML Capabilities - Beta
+We are happy to announce we now offer SAML capabilities! Contact us if you want to try it. 
+[Learn More](https://insights.docs.fairwinds.com/configure/management/membership/#single-sign-on-beta)
```

Comment:
```suggestion
[Learn More](/configure/management/membership/#single-sign-on-beta)
```

If we make it a relative link like this it'll be easier to tell if it breaks in the future.